src/main/java/Duke.java

```diff
@@ -1,10 +1,68 @@
+import duke.command.Command;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+import duke.parser.Parser;
+import duke.storage.Storage;
+
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+
+/**
+ * A text based chatbot able to keep track of tasks.
+ */
 public class Duke {
+    public boolean isListening = true;
+    private final String FILE_PATH;
```

Comment:
Good to see you're following the naming convention for constants! Would setting this to a pre-defined, hardcoded value be better than defining in your `main` method?